src/parser.js

```diff
@@ -34,99 +46,171 @@ class Parser {
    * @param {DOM} tree - DOM tree or Node
    * @param {Number} level - Current tree level
    */
-  * walk (tree, level) {
+  *walk(tree, level) {
     if (!tree) {
       return
     }
 
     for (let i = 0; i < tree.length; i++) {
```

Comment:
why `let` here and const in the `for` block, line 85

Reply:
`const` would be invalid here due to the value change `i++`. Works well in the `for...of` (line 85) because the value never changes.

Reply:
ah, cool